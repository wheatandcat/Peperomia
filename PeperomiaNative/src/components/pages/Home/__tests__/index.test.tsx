import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Index from '../index';

describe('components/pages/Home/index.tsx', () => {
  let wrapper: ShallowWrapper;

  const propsData: any = () => ({});

  beforeEach(() => {
    wrapper = shallow(<Index {...propsData()} />);
  });

  it('正常に表示されている', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
