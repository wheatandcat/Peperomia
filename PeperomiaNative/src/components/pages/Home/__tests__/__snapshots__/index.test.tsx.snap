// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/pages/Home/index.tsx 正常に表示されている 1`] = `
<StackNavigator
  initialRouteName="Home"
>
  <Screen
    component={[Function]}
    initialParams={
      Object {
        "refresh": false,
      }
    }
    name="Home"
    options={[Function]}
  />
  <Screen
    component={[Function]}
    name="Schedule"
    options={[Function]}
  />
</StackNavigator>
`;
